staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -1734,6 +1759,30 @@ type VolumeProjection struct {
 	// serviceAccountToken is information about the serviceAccountToken data to project
 	// +optional
 	ServiceAccountToken *ServiceAccountTokenProjection `json:"serviceAccountToken,omitempty" protobuf:"bytes,4,opt,name=serviceAccountToken"`
+
+	// ClusterTrustBundle allows a pod to access the `.spec.trustBundle` field
+	// of a ClusterTrustBundle object in an auto-updating file.
+	//
+	// Alpha, gated by the ClusterTrustBundleProjection feature gate.
+	//
+	// ClusterTrustBundle objects can either be selected by name, or by the
+	// combination of signer name and a label selector.
+	//
+	// When selecting by name, the referenced ClusterTrustBundle object must
+	// have an empty spec.signerName field.
+	//
+	// When selecting by signer name, the contents of all ClusterTrustBundle
+	// objects associated with the signer and matching the label will be unified
+	// and deduplicated.
+	//
+	// Kubelet performs aggressive normalization of the PEM contents written
+	// into the pod filesystem.  Esoteric PEM features such as inter-block
+	// comments and block headers are stripped.  Certificates are deduplicated.
+	// The ordering of certificates within the file is arbitrary, and Kubelet
+	// may change the order over time.
+	//
+	// +optional
```

Comment:
There's a featureGate tag you can add: https://github.com/kubernetes/kubernetes/blob/6f4783b787bcdf1173c0089c61277a0871d8bad1/staging/src/k8s.io/api/core/v1/types.go#L545

Reply:
Done.